Scripts/gen_FlatMuDaugther.sh

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/bash
+#
+# create fcl for producing primaries from stopped particles
+# starting from a given primary that allows to set:
+# pdgID, startMom and endMom. 
+# Tested with the FlateMinus/Plus primaries
+#
+# this script requires mu2etools and dhtools be setup
+#
+# $1 is the primary
+# $2 is the production (ie MDC2020)
+# $3 is the stops production version
+# $4 is the output primary production version
+# $5 is the kind of input stops (Muminus, Muplus, IPAMuminus, IPAMuplus, or Cosmic)
+# $6 is the number of jobs
+# $7 is the number of events/job
+# $8 is the pdgId of the paritcle to generate
+# $9 is the startMom
+# $10 is the endMom
+# $11 is the name of the BField file
+#
+# example for running this script:
+#        source Production/Scripts/gen_FlatMuDaugther.sh FlatePlus MDC2020 k v01 Muminus 2 100 -11 30 100 "Offline/Mu2eG4/geom/bfgeom_DS70_no_tsu_ps_v01.txt"
```

Comment:
Daugther is misspelled here and in the filename

Reply:
Also, can you replace `v01` with our convention, for example `s`? particle is also misspelled